src/lib/y2network/autoinst/interfaces_reader.rb

```diff
@@ -20,8 +20,12 @@
 require "yast"
 require "y2network/dns"
 require "y2network/autoinst/type_detector"
+require "y2storage/autoinst_issues/invalid_value"
+require "y2storage/autoinst_issues/missing_value"
```

Comment:
Although the y2storage classes fits better to our needs, @imobachgs considers that is better to use the yast2 classes and try to adapt and unify them instead of reusing the storage ones and I agree with it.

As commented with him, the y2storage ones fits perfectly to our needs and do not wanted to define our own classes by now, but it is better to not add another level of dependency.